.github/workflows/lint.yml

```diff
@@ -23,4 +23,4 @@ jobs:
       - name: Lint
         uses: golangci/golangci-lint-action@v3.4.0
         with:
-          version: v1.52
+          version: v1.54
```

Comment:
Bumped this because it seemed to resolve some linting errors. cli/cli is already on 1.54+